code/game/gamemodes/clock_cult/clock_structures/ocular_warden.dm

```diff
@@ -134,3 +134,14 @@
 	target = null
 	visible_message("<span class='warning'>[src] settles and seems almost disappointed.</span>")
 	return 1
+
+/obj/structure/destructible/clockwork/ocular_warden/get_efficiency_mod(increasing)
+	if(GLOB.ratvar_awakens)
+		return increasing ? 0.5 : 2
+	. = 1
+	for(var/turf/T in getline(src, target))
+		for(var/obj/structure/O in T)
+			if(O.density)
+				. -= 0.15
+	. -= (get_dist(src, target) * 0.05)
+	. = max(., 0.1) //The lowest damage a warden can do is 10% of its normal amount (0.25 by default)
```

Comment:
you have to support the argument for the entire thing.

Reply:
I can't actually tell what the argument does, there's no real documentation on it.

Reply:
It gives the reciprocal of the normal return value.

add these lines to the end
```
if(increasing)
    . = 1/.
```

Reply:
If true, it provides an increasing multiplier instead of a decreasing one, so that if used on values where lower is better it works properly.